### PR TITLE
MIP21: Cure before tell

### PR DESCRIPTION
**Pull request by e18r**
~The test `testSpellIsCast_RWA001_INTEGRATION_CURE` should fail. However, it succeeds for an unknown reason.~ Ready for review

### CODE REVIEW

File: src/test/RwaSpell.t.sol [hunk 1]
```diff
@@ -804,6 +804,20 @@ contract DssSpellTest is DSTest, DSMath {
         assertTrue(oracle.good("RWA001-A"));
     }
 
+    function testSpellIsCast_RWA001_INTEGRATION_CURE() public {
```
Comment (gbalabasquer) [inline src/test/RwaSpell.t.sol]: You need to redeploy everything in order to take the changes you are doing in `cure`, then this test will fail.

Comment (gbalabasquer) [reply src/test/RwaSpell.t.sol]: Otherwise it is calling a `oracle` that doesn't implement the new constraints in `cure`.

Comment (godsflaw) [reply src/test/RwaSpell.t.sol]: Same comment here as the other PR too: we need to merge upstream back into this and update `vote()` and `voteTemp()`.

**Review by godsflaw — CHANGES_REQUESTED** [review]

**Comment by e18r** [comment]
@gbalabasquer still passing after redeploying. What am I missing?

**Comment by gbalabasquer** [comment]
Fixed

**Review by godsflaw — APPROVED** [review]

**Pull request merged**